test/system/070-build.bats

```diff
@@ -145,7 +135,7 @@ echo "\$1"
 printenv | grep MYENV | sort | sed -e 's/^MYENV.=//'
 EOF
 
-    # For overriding with --env-file
+    # For overridding with --env-file
```

Comment:
Typo introduced: "overriding" was/is correct

Reply:
IDE dictionary or spelling check is broken on that word.  I'll revert.